dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -212,9 +216,9 @@ export default function Catalog() {
   // We do not currently support package repositories on additional clusters.
   const supportedCluster = cluster === kubeappsCluster;
   useEffect(() => {
-    if (!supportedCluster || namespace === globalReposNamespace) {
-      // Global namespace or other cluster, show global repos only
-      dispatch(actions.repos.fetchRepos(globalReposNamespace));
+    if (!namespace || !supportedCluster || namespace === globalReposNamespace) {
+      // All Namespaces. Global namespace or other cluster, show global repos only
+      dispatch(actions.repos.fetchRepos(""));
```

Comment:
```suggestion
      dispatch(actions.repos.fetchRepos(globalReposNamespace));
```

If I'm not mistaken the result will be the same, but isn't it easier to read that the action is retrieving global repos?

Reply:
I've checked it and, in the current impl, fetching `globalReposNamespace` instead of just `""` only gives us the global repos, not every repo the user can access to:

![image](https://user-images.githubusercontent.com/11535726/177784521-789551db-79b3-4e7f-96ed-dbdb2143bded.png)